_plugins/jekyll_get.rb

```diff
@@ -0,0 +1,39 @@
+require 'json'
+require 'hash-joiner'
+require 'open-uri'
+
+module Jekyll_Get
+  class Generator < Jekyll::Generator
+    safe true
+    priority :highest
+
+    def generate(site)
+      config = site.config['jekyll_get']
+      if !config
```

Comment:
Just as a way to disable without deleting the config? We could do that, but let's not hold this up any further for discussions about the plugin. The goal was to validate that it solves our problem (getting data from the hub on build). Since that seems answered, we should get this plugin into a separate repo where we can improve it with additional ideas like this.